server/routes/induction/common/inductionController.ts

```diff
@@ -22,4 +22,12 @@ export default abstract class InductionController {
     const updatedPageFlowHistory = setCurrentPage(req.session.pageFlowHistory, currentPageUri)
     req.session.pageFlowHistory = updatedPageFlowHistory
   }
+
+  /**
+   * Returns `true` if the previous page in the Page Flow History was Check Your Answers
+   */
+  previousPageWasCheckYourAnswers(req: Request): boolean {
+    const { pageFlowHistory } = req.session
+    return pageFlowHistory && getPreviousPage(pageFlowHistory).endsWith('/check-your-answers')
```

Comment:
Because I've used `endsWith` I think this method will also work for checking whether the previous page was Check Your Answers for the Create journey as and when we get there (the Update and Create journeys will have different paths, but will end with the same page name/path element)